statsmodels/tsa/tests/test_holtwinters.py

```diff
@@ -0,0 +1,116 @@
+        #aust = pd.read_json(aust_json, typ='Series').sort_index()
+        data = [41.727457999999999, 24.04185, 32.328102999999999, 37.328707999999999, 46.213152999999998, 29.346326000000001, 36.482909999999997, 42.977719, 48.901524999999999, 31.180221, 37.717880999999998, 40.420211000000002, 51.206862999999998, 31.887228, 40.978262999999998, 43.772491000000002, 55.558566999999996, 33.850915000000001, 42.076383, 45.642291999999998, 59.766779999999997, 35.191876999999998, 44.319737000000003, 47.913736]
+        index= ['2005-01-01 00:00:00', '2005-04-01 00:00:00', '2005-07-01 00:00:00', '2005-10-01 00:00:00', '2006-01-01 00:00:00', '2006-04-01 00:00:00', '2006-07-01 00:00:00', '2006-10-01 00:00:00', '2007-01-01 00:00:00', '2007-04-01 00:00:00', '2007-07-01 00:00:00', '2007-10-01 00:00:00', '2008-01-01 00:00:00', '2008-04-01 00:00:00', '2008-07-01 00:00:00', '2008-10-01 00:00:00', '2009-01-01 00:00:00', '2009-04-01 00:00:00', '2009-07-01 00:00:00', '2009-10-01 00:00:00', '2010-01-01 00:00:00', '2010-04-01 00:00:00', '2010-07-01 00:00:00', '2010-10-01 00:00:00']
+        aust = pd.Series(data, index)
+        aust.index = pd.DatetimeIndex(aust.index, freq=pd.infer_freq(aust.index))
+        cls.aust = aust
+
+    def test_predict(self):
+        fit1 = ExponentialSmoothing(self.aust, m=4, trend='add', seasonal='mul').fit()
+        fit2 = ExponentialSmoothing(self.aust, m=4, trend='add', seasonal='mul').fit()
+#        fit3 = ExponentialSmoothing(self.aust, m=4, trend='add', seasonal='mul').fit(remove_bias=True, use_basinhopping=True)
+        assert_almost_equal(fit1.predict('2011-01-01 00:00:00', '2011-10-01 00:00:00'), [61.3083,37.3730,46.9652,51.5578], 3)
+        assert_almost_equal(fit2.predict(end='2011-10-01 00:00:00'), [61.3083,37.3730,46.9652,51.5578], 3)
+#        assert_almost_equal(fit3.predict('2010-10-01 00:00:00', '2010-10-01 00:00:00'), [49.087], 3)
+
+        
+    def test_ndarray(self):
+        fit1 = ExponentialSmoothing(self.aust.values, m=4, trend='add', seasonal='mul').fit()
+        assert_almost_equal(fit1.forecast(4), [61.3083,37.3730,46.9652,51.5578], 3)
+        
+    def test_forecast(self):
+        fit1 = ExponentialSmoothing(self.aust, m=4, trend='add', seasonal='add').fit()
+        assert_almost_equal(fit1.forecast(steps=4), [60.9542,36.8505,46.1628,50.1272], 3)
+        
+    def test_simple_exp_smoothing(self):
+        fit1 = SimpleExpSmoothing(self.oildata_oil).fit(0.2,optimized=False)
+        fit2 = SimpleExpSmoothing(self.oildata_oil).fit(0.6,optimized=False)
+        fit3 = SimpleExpSmoothing(self.oildata_oil).fit()
+        assert_almost_equal(fit1.forecast(1), [484.802468], 4)
```

Comment:
in general we prefer no `assert_allclose` because it is more precise and flexible to set atol and rto instead of decimals.
However, assert_almost_equal looks appropriate here with fixed print precision in reference numbers.